x11-packages/rofi/source-modes-combi.c.patch

```diff
@@ -0,0 +1,21 @@
+diff '--color=auto' -uNr rofi-1.7.4/source/modes/combi.c rofi-1.7.4.mod/source/modes/combi.c
+--- rofi-1.7.4/source/modes/combi.c	2022-07-28 04:08:02.000000000 +0530
++++ rofi-1.7.4.mod/source/modes/combi.c	2022-08-19 10:27:45.192796288 +0530
+@@ -142,6 +142,17 @@
+     mode_set_private_data(sw, NULL);
+   }
+ }
++
++#if defined(__ANDROID_API__) && __ANDROID_API__ < 24
```

Comment:
Do we need this? We only support API 24 and up now.

Reply:
> Do we need this? We only support API 24 and up now.

Same idea, termux for Android 5 does not have X11 repo for the time being.

Reply:
So the patch isn't needed?

Reply:
@sylirre do we still need this?

Reply:
Remove it